ddtrace/commands/ddtrace_run.py

```diff
@@ -23,13 +23,13 @@
 
```

Comment:
This is a bash syntax, it might be different in other shells and it's actually of poor use if you `exec` the program.